test/integration/bodyFormDataRest.spec.ts

```diff
@@ -0,0 +1,64 @@
+import BodyFormDataClient, {
+  BodyFormDataRestClient
+} from "./generated/bodyFormDataRest/src";
+import { assert } from "chai";
+
+describe("Integration tests for BodyFormData", () => {
+  let client: BodyFormDataRestClient;
+
+  it("should correctly accept file via form", async function() {
+    client = BodyFormDataClient({ allowInsecureConnection: true });
+
+    const fileName: string = `sample.png`;
+    const content = new Uint8Array([
+      72,
+      101,
+      108,
+      108,
+      111,
+      32,
+      87,
+      111,
+      114,
+      108,
+      100
+    ]);
+
+    const result = await client.path("/formdata/stream/uploadfile").post({
+      contentType: "multipart/form-data",
+      body: { fileContent: content, fileName },
```

Comment:
`{ fileContent: content, fileName: fileName }` missed a fileName key ?

Reply:
If the key has the same name as a variable there is no need to add key value. For example

```typescript
let foo = "foo"

console.log({ foo: foo })
// output:
// {
//  "foo": "foo"
// } 


console.log({ foo})
// output:
// {
//  "foo": "foo"
// } 
```